### ISSUE TITLE
How to enter third party mixing parameters when have a pre-v10 fluid file?

### ISSUE THREAD

**Issue by kjschultz527** [usnistgov/REFPROP-issues#110]
### Description

I have successfully entered mixing parameters supplied by an outside party for binaries that have version 10 fluid files -- just replace the CASNO identifier used previously with the fluid's new HASH code.  But what if I have a fluid file provided by an outside party that was generated for an earlier version of REFPROP (v8 in this case - I can call the fluid file by itself just fine)?  What identifier do I use for HASH?  Tried to use HASH for Fluid1 (v10.fld) / CAS for Fluid2 (v8.fld), but REFPROP did not find the MPs. Tried to use CAS for Fluid1 / CAS for Fluid2 - didn't work either.  (REFPROP did find MPs downstream of where I tried to insert the new MPs, so I don't think there is a format problem.  Can't provide code here because of confidentiality.)

### Versions

**REFPROP Version:** REFPROP version 10
**Operating System and Version:** Window 10
**Access Method:** REFPROP GUI, Excel

**Comment by ianhbell** [comment]
N.B.: I have made minor edits to your question to tidy it up

**Comment by EricLemmon** [comment]
When we developed version 10, we made sure that it was backwards compatible with all fluid files and the mixture file hmx.bnc.  But reading your note scared me that perhaps we broke that feature somewhere along the way before releasing version 10!  So I just tried this.  I made up a mixture of nitrogen+ethane and calculated a VLE point.  I then closed Refprop, edited the ethane.fld file, removed the lines with the hash number and the alternate hash number, and then edited the hmx.bnc file and changed the hash number to the CAS number for ethane as shown below:

`?Nitrogen/Ethane                                           [N2/C2]`
`?Kunz and Wagner (2007)`
`  edbdc8e0/74-84-0`
`    KW6      1.007671428    1.098650964    0.978880168    1.042352891    1.             0.             0. 0. 0. 0. 0. 0.`

I restarted the Refprop graphical interface, made up the same mixture, and calculated the same point.  The results were the same.  Is this the same that you tried?

**Comment by kjschultz527** [comment]
Yes, it is the same as I tried.  I gave it another try today -- and it worked; that is, View Mixing Parameters displayed the correct values.  Apparently, I forgot to close REFPROP and then restart it after editing the HMX.BNC -- sorry.  (But this indicates that I can employ the same strategy that I used with v9 to track and use multiple sets of mixing parameters for a particular binary, but hopefully won't have nearly as many now that NIST has "standardized" many containing the new HFOs.)
